apps/web/src/components/Advisory.tsx

```diff
@@ -0,0 +1,13 @@
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const Advisory: React.FC = ({ children }) => {
+  return (
+    <div className='bg-bcLightBlueBackground py-4  text-left flex items-center'>
```

Comment:
the `rounded` class should be added here

Reply:
I thought about that, but the design doesn't have rounded corners.

Reply:
I see rounded on the version of designs I have 🤔  
![image](https://user-images.githubusercontent.com/71518072/148105249-e5ec0296-b36c-4daa-a0ef-b15a86f81187.png)
https://www.figma.com/file/Cf5M3yWTLdJxngU6l6o17c/EHPR---Mockups?node-id=13%3A1211

Reply:
I see what happened, I had the component selected in figma, so it looked like it had short corners. It does feel a bit weird that this has rounded corners but the component above as sharp corners. 🤷 
 
<img width="121" alt="Screen Shot 2022-01-04 at 10 20 47 AM" src="https://user-images.githubusercontent.com/54554766/148105742-5339b9ea-802e-4c76-b7f3-cbb6e195847c.png">